### ISSUE TITLE
Re-init A11y module when `children` changes

### ISSUE THREAD

**Issue by MorevM** [nolimits4web/swiper#5692]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/swiper-infinite-loop-vue-forked-vlfpw6?file=/src/App.vue

### Bug description

Steps to reproduce: 

1) Click on the `Prepend` or `Append` button
2) Check aria attributes are missed on newly added slides

It's even worse with `Shuffle` button - aria labels shuffled as well.

### Expected Behavior

Slides should have correct `aria-label`'s after `children` update

### Actual Behavior

Module respects only initially set aria attributes and doesn't react to changes.

Seems it's needed to expose A11y module on Swiper params the same way as Pagination/Navigation does and do extra check for it in https://github.com/nolimits4web/swiper/blob/master/src/vue/update-swiper.js and re-init the module if `children` changed.

Most likely it doesn't work correctly for other frameworks as well, but I have time/interest only for Vue version.
I can do PR for Vue version if bug is really accepted.

### Swiper version

8.1.4

### Platform/Target and Browser Versions

Not platform/browser specific

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR
